common/types.go

```diff
@@ -580,3 +601,19 @@ func (l Location) ContainsAddress(a Address) bool {
 func (l Location) Equal(cmp Location) bool {
 	return bytes.Equal(l, cmp)
 }
+
+func (l Location) CommonDom(cmp Location) Location {
```

Comment:
Same here. It would be nice to have a comment. Little hard to follow the logic without context.

Reply:
Sure thing,  I'll add a comment here too. See my response to your other question for an explanation of "common dom"

Reply:
thanks